packages/rich-text/src/component/index.native.js

```diff
@@ -708,8 +726,11 @@ export class RichText extends Component {
 			value = '';
 		}
 		// On android if content is empty we need to send no content or else the placeholder will not show.
-		if ( ! this.isIOS && value === '' ) {
-			return value;
+		if (
+			! this.isIOS &&
+			( value === '' || value === EMPTY_MULTILINE_PARAGRAPH )
+		) {
+			return '';
```

Comment:
This insures that empty paragraph tags do not get passed to the RCTAztecView (thus hiding the placeholder)